plugins/arm_a64/a64.ml

```diff
@@ -0,0 +1,23 @@
+open Core_kernel
```

Comment:
Please, indent the code using ocp-indent. Which development environment are you using? It shall be set up so that it automatically indents your code. Tell us what you're using and we will help you in setting things up.

Reply:
I use vscode.

Reply:
you can now just do `make indent` and it will indent your code (provided that you have ocp-indent installed)

Reply:
Sorry for that. I am about to do it this evening. So I just need to, with ocp-indent installed, run `make indent` without modifying the Makefile?

Reply:
Yes, just `git pull` your branch to get the latest master (I've already updated the PR)

Reply:
ok